lib/matplotlib/tests/test_colors.py

```diff
@@ -395,22 +395,48 @@ def test_TwoSlopeNorm_premature_scaling():
 
 
 def test_SymLogNorm():
-    """
-    Test SymLogNorm behavior
-    """
-    norm = mcolors.SymLogNorm(3, vmax=5, linscale=1.2)
+    # Test SymLogNorm behavior
+    norm = mcolors.SymLogNorm(linthresh=3, vmax=5, linscale=1.2, base=np.e)
     vals = np.array([-30, -1, 2, 6], dtype=float)
     normed_vals = norm(vals)
-    expected = [0., 0.53980074, 0.826991, 1.02758204]
+    expected = [-0.842119, 0.450236, 0.599528, 1.277676]
     assert_array_almost_equal(normed_vals, expected)
     _inverse_tester(norm, vals)
     _scalar_tester(norm, vals)
     _mask_tester(norm, vals)
 
-    # Ensure that specifying vmin returns the same result as above
-    norm = mcolors.SymLogNorm(3, vmin=-30, vmax=5, linscale=1.2)
+
+def test_symlognorm_vals():
+    vals = [-10, -1, 0, 1, 10]
+
+    norm = mcolors.SymLogNorm(linthresh=1, vmin=-10, vmax=10, linscale=1)
     normed_vals = norm(vals)
-    assert_array_almost_equal(normed_vals, expected)
+    expected = [0, 0.25, 0.5, 0.75, 1]
```

Comment:
I don't think this is consistent with `SymmetricLogTransform` yet.  I'm OK if we want to change that as well to be consistent with this code, but they can't be inconsistent!  

```python 
import matplotlib.scale as mscale

trans = mscale.SymmetricalLogTransform(10, 1, 1)
new = trans.transform([-10, -1, 0, 1, 10])
new = (new - new[0]) / (new[-1] - new[0])
print(new)
```

```
[0.0 0.23684210526315788 0.5 0.7631578947368421 1.0]
```

I'm fine if this implementation is desired, but then we need to change `SymmetricLogTransform`.